buffer/buffer.go

```diff
@@ -99,6 +103,16 @@ func New(next http.Handler, setters ...optSetter) (*Buffer, error) {
 	return strm, nil
 }
 
+// Logger defines the logger the forwarder will use.
```

Comment:
Stray mention of "forwarder" here, should be "buffer" or something similar.

Reply:
Ah, yeah copy-paste errors.